Journal of Extension.js

```diff
@@ -0,0 +1,360 @@
+{
+	"translatorID": "7e56f792-e96b-42a1-913c-40c69cc9ce40",
+	"label": "Journal of Extension",
+	"creator": "John Weis",
+	"target": "^http?://www.joe.org/((index|\\w+\\d).php)?",
```

Comment:
you want ```^https?``` in the beginning. Also, escape periods (```www\\.joe\\.org``` etc).
Finally, if you make the entire last part optional (the ?) why include it in the target at all. Ideally we'd just require it.